src/sentry/api/bases/organization_events.py

```diff
@@ -43,6 +44,7 @@
     "metrics": metrics_performance,
     "profiles": profiles,
     "issuePlatform": issue_platform,
+    "functions": functions,
```

Comment:
nit: but these are specifically profiling functions right? Could we make this a bit more verbose just in case?

Reply:
we could name it `profile_functions` to make it explicit this is a profiling related dataset